package.json

```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "node_modules/.bin/eslint *.js lib/*.js test/",
-    "test": "npm run lint && mocha --check-leaks"
+    "pretest": "npm run lint",
+    "test": "mocha --check-leaks"
   },
```

Comment:
I think it's better to leave them separated, otherwise they can become unreadable

Reply:
Not sure I follow you. Did you mean combining them back? Anyway, both options look good to me.

Reply:
oh sorry I thought it was the other way around)